cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -82,36 +111,38 @@ metadata:
   name: token-${safe_username}
 type: Opaque
 EOF
-  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" &
+  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" "${SYSTEM_NAMESPACE}" &
```

Comment:
I believe this comment is no longer applicable. (Just documenting in case someone is looking at the same view I am).

We no longer handle any updates in this script, so we're just creating the kube-system token here.